src/components/Datepicker/components/DatepickerWeek.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <div class="sb-datepicker-week">
+    <span v-for="weekLabel in weekLabels" :key="weekLabel">
```

Comment:
Wouldn't be better to use an index variable as :key?
Like `<span v-for="(weekLabel, i) in weekLabels" :key="i">`

Reply:
Also, what do you think about using "weekDays" instead of "weekLabels"?

Reply:
That's a good name. And about the use of keys, I used the own name as a key because each name is unique, so I don't think is necessary to use the key in the array if the each value is unique. Understand what I mean?

Reply:
If you are going to use v-for this way `v-for="weekLabel in weekLabels" :key="weekLabel"`, prefer to use `:key="weekLabel.id"`, because **v-for** implements an `id` internally 😄

Reply:
No, it's not necessary because the `weekLabels` (now `weekDays`) property is only an array of strings. [In according to documentation](https://vuejs.org/v2/guide/list.html#Maintaining-State), I can use strings or numbers as keys in `v-for`, therefore, I can use the item in the array because it's a string.